src/main/java/org/pac4j/vertx/handler/impl/Pac4jAuthHandlerOptions.java

```diff
@@ -6,9 +6,7 @@
 public class Pac4jAuthHandlerOptions {
 
   private final String clientName;
-  private boolean isAjax = false;
-  private String requireAnyRole = "";
-  private String requireAllRoles = "";
+  private boolean allowDynamicClientSelection = false;
```

Comment:
Sorry for the round-trip here: I will remove `allowDynamicClientSelection` and `useSessionForDirectClient`.

I will have only two parameters left: `clientName` (a list of clients) and `authorizerName` (a list of authorizers).